Organization/Organization/MainViewController.h

```diff
@@ -1,11 +1,15 @@
 #import <UIKit/UIKit.h>
 #import "SavedProtocol.h"
+#import <CoreData/CoreData.h>
 
 @class Employee;
 @class Organization;
 
-@interface MainViewController : UITableViewController<SaveEmployeeProtocol>
+@interface MainViewController : UITableViewController<UITableViewDataSource, SaveEmployeeProtocol>
 
 @property (strong, nonatomic) Organization *org;
+@property (readonly, strong, nonatomic) NSManagedObjectModel *managedObjectModel;
```

Comment:
Why do you have multiple models, contexts and coordinators? They are already defined in AppDelegate.